plugin.js

```diff
@@ -61,8 +63,21 @@ function onReqHandlerWrapper (fastify) {
   }
 }
 
+function getHook (hook, next) {
+  const hooks = {
```

Comment:
nitpick: Would have used any array here.

Reply:
yes, but accessing the object is a bit faster :)